### PR TITLE
Ship shared files with absolute path

### PR DESCRIPTION
**Pull request by julmon**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,6 +39,16 @@
             'share/users',
         ]),
         ('lib/systemd/system', ['packaging/temboard-agent.service']),
+        # Installation from sources
+        ('/usr/share/temboard-agent/', [
```
Comment (bersace) [inline setup.py]: This breaks installation is userspace (e.v. `pip install --user` or in virtualenv).

Comment (bersace) [reply setup.py]: Cf. http://readthedocs.org/projects/temboard-agent/builds/6440200/

Comment (julmon) [reply setup.py]: Thank you. How to ship them properly then ?

**Comment by bersace** [comment]
https://circleci.com/gh/dalibo/temboard-agent/1293

```
tests_centos7_pg10_1  |         if ret_code != 0:
tests_centos7_pg10_1  | >           raise Exception(str(stderr))
tests_centos7_pg10_1  | E           Exception: createuser: could not connect to database postgres: could not connect to server: No such file or directory
tests_centos7_pg10_1  | E           	Is the server running locally and accepting
tests_centos7_pg10_1  | E           	connections on Unix domain socket "/tmp/tests_temboard/1513777803794/pg/run/.s.PGSQL.5445"?
tests_centos7_pg10_1  | 
tests_centos7_pg10_1  | test/legacy/test/temboard.py:111: Exception
tests_centos7_pg10_1  | ===================== 93 passed, 4 error in 56.08 seconds ======================
tests_centos7_pg10_1  | + teardown
tests_centos7_pg10_1  | + exit_code=1
```

Any idea ?

**Comment by julmon** [comment]
Yes, race condition. Postgres not yet started when the test began.

File: setup.py (hunk 1 above)
Comment (julmon) [reply setup.py]: Alright, we do not need absolute path when installing from sources using `pip install .`. Let's go that way.

**Pull request merged**